templates/part.board.mainView.php

```diff
@@ -28,7 +28,7 @@
 	</form>
 	</div>
 </div>
-<div id="board" class="scroll-container">
+<div id="board" class="scroll-container" ng-click="sidebar.show=false">
```

Comment:
This will break when you try to open up the same card again. Please use ui-sref to navigate to the board route. 

See: https://github.com/nextcloud/deck/blob/master/templates/part.card.php#L9